src/components/screen/SearchUser.js

```diff
@@ -179,77 +161,45 @@ class Screen extends Component<Props, State> {
-                <Animated.View style={[
-                  styles.viewSearch,
-                  {
-                    height: 50,
-                    transform: [{
-                      translateY: this.scrollY.interpolate({
-                        inputRange: [-50, 0, 50, 100],
-                        outputRange: [0, 0, -50, -50],
-                      })
-                    }],
-                  }
-                ]}>
-                  <Animated.View style={{
-                    position: 'absolute',
-                    width: '100%',
-                    paddingHorizontal: 20,
-                    height: 50,
-                    opacity: this.scrollY.interpolate({
-                      inputRange: [-50, 0, 50, 100],
-                      outputRange: [1, 1, 0, 0],
-                    })
-                  }}>
-                    <TextInput
+        { (data) => {
+          return (
+            <StyledSafeAreaView>
+              <StyledContainer>
+                <StyledSearchView>
+                  <StyledTextInputWrapper>
+                    <StyledTextInput
+                      id="styledInput"
                       onChangeText={(text) => this.onTxtChanged(text)}
                       underlineColorAndroid='transparent' // android fix
                       autoCapitalize='none'
                       autoCorrect={false}
                       multiline={false}
                       // value={this.searchTxt}
-                      style={{
-                        width: '100%',
-                        height: 30,
-                        top: 10,
-                        backgroundColor: 'rgb(247,248,251)',
-                        borderRadius: 4,
-                        paddingLeft: 34,
-                        paddingRight: 10
-                      }}
-                      // onSubmitEditing={this.onSearch}
                       defaultValue={''}
                     />
-                    <Image source={IC_SEARCH} style={styles.imgSearch}/>
-                  </Animated.View>
-                </Animated.View>
-                <AnimatedFlatList
-                  id='animated'
+                    <StyledSearchImage source={IC_SEARCH} />
+                  </StyledTextInputWrapper>
+                </StyledSearchView>
+                <StyledAnimatedFlatList
+                  id='animatedFlatlist'
                   style={{
-                    width: '100%',
-                    height: '100%',
-                    marginBottom: -50,
                     transform: [{
                       translateY: this.scrollY.interpolate({
-                        inputRange: [-50, 0, 50, 100],
-                        outputRange: [0, 0, -50, -50],
+                        inputRange: [0, 50, 100],
+                        outputRange: [0, 25, 0],
                       })
-                    }],
+                    }]
                   }}
                   contentContainerStyle={this.containerStyle}
```

Comment:
name the render stuff like `this.renderContainer()`. Naming looks bit ambiguous.

Reply:
got it 😅